lib/recurly/purchase.php

```diff
@@ -54,11 +54,43 @@ public static function preview($purchase, $client = null) {
    * @param Recurly_Client $client Optional client for the request, useful for mocking the client
    * @return object Recurly_InvoiceCollection
    * @throws Recurly_Error
+   *
+   * Note: To use this endpoint, you may have to contact Recurly support to have it enabled on your subdomain.
```

Comment:
Good note to add here

Reply:
Yeah, I had to email them because I was getting an error that said something like: feature not enabled.  I have found very little on google concerning these endpoints.  Seems to be working okay now.